packages/react-components/react-search/stories/SearchBox/SearchBoxAppearance.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { SearchBox } from '@fluentui/react-search';
+import { makeStyles, mergeClasses, shorthands, tokens, useId, Label } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    maxWidth: '490px',
+  },
+  field: {
+    display: 'grid',
+    gridRowGap: tokens.spacingVerticalXXS,
+    marginTop: tokens.spacingVerticalMNudge,
+    ...shorthands.padding(tokens.spacingHorizontalMNudge),
+  },
+  filledLighter: {
+    backgroundColor: tokens.colorNeutralBackgroundInverted,
+    '> label': {
+      color: tokens.colorNeutralForegroundInverted2,
+    },
```

Comment:
We discourage the use of selectors in the examples, instead you could add a label
```tsx
invertedLabel: {
  color: tokens.colorNeutralForegroundInverted2,
}
```

and pass it in the slot's classname. The example I linked to before has that implementation.